Source/WebCore/rendering/RenderLayerInlines.h

```diff
@@ -51,4 +52,19 @@ inline bool RenderLayer::paintsWithTransparency(OptionSet<PaintBehavior> paintBe
     return (paintBehavior & PaintBehavior::FlattenCompositingLayers) || !isComposited();
 }
 
+inline bool RenderLayer::hasNonOpacityTransparency() const
+{
+    if (renderer().hasMask())
+        return true;
+
+    if (hasBlendMode() || isolatesBlending())
+        return true;
+
+    // SVG clip-paths may use clipping masks, if so, flag this layer as transparent.
```

Comment:
Oops, this is also LBSE only, so this even needs the check condition if LBSE is enabled at run-time (via document().settings()...).

Reply:
Right, fixed.